contracts/assemblyscript/nep4-basic/__tests__/main.unit.spec.ts

```diff
@@ -58,23 +59,23 @@ describe('revoke_access', () => {
   })
 })
 
-describe('transfer', () => {
-  it('allows owner to transfer given `token_id` to given `owner_id`', () => {
+describe('transfer_from', () => {
+  it('allows owner to transfer_from given `token_id` to given `owner_id`', () => {
```

Comment:
small nits: 

* if we keep `describe('transfer_from'` first, the diff here will be smaller & easier to understand
* I'd prefer that the test output read like a plain-English specification – to me, this is the purpose of having these "spec"-style test DSLs. To that end, I find the use of `transfer_from` in the `it` lines strange. Currently, the "executable specification" here reads as follows:

  > transfer
  >
  > * allows owner to transfer_from given `token_id` to given `owner_id`
  > * allows escrow to transfer_from given `token_id` to given `new_owner_id` if `owner_id` matches
  > * prevents escrow from using transfer_from given `token_id` to given `new_owner_id if `owner_id` does not match`
  > * prevents anyone else from using transfer_from on the token